### PR TITLE
feat: Added support for django3.2

### PR DESCRIPTION
**Pull request by Jawayria**
JIRA: https://openedx.atlassian.net/browse/BOM-2622

### CODE REVIEW

**Comment by Jawayria** [comment]
Running the tests raises this warning and it wasn't fixed under the scope of this PR
`RemovedInDjango41Warning: 'celery_utils' defines default_app_config = 'celery_utils.apps.CeleryUtilsConfig'. Django now detects this configuration automatically. You can remove default_app_config.`

**Comment by mraarif** [comment]
@Jawayria which version of Django exactly has added this support?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,43 +1,43 @@
...
-         with:
-           python-version: ${{ matrix.python-version }}
+    steps:
+    - uses: actions/checkout@v1
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
 
-       - name: Install pip
```
Comment (mraarif) [inline .github/workflows/ci.yml:27]: can we undo these indentation changes

File: tox.ini [hunk 2]
```diff
@@ -18,51 +18,52 @@ addopts = --cov celery_utils --cov-report term-missing --cov-report xml
...
+    -r{toxinidir}/requirements/test.txt
+commands = 
+    py.test tests/ celery_utils/ {posargs}
 
 [testenv:docs]
-setenv =
-	DJANGO_SETTINGS_MODULE = test_settings
-	PYTHONPATH = {toxinidir}
-whitelist_externals =
-	make
```
Comment (mraarif) [inline tox.ini:35]: same here

**Review by mraarif — APPROVED** [review]

**Review by mraarif — APPROVED** [review]

**Review by mraarif — APPROVED** [review]

**Review by awais786 — APPROVED** [review]

**Pull request merged**